build/fbcode_builder/getdeps/builder.py

```diff
@@ -184,6 +184,11 @@ def run_tests(
         cmd = ["make"] + self.test_args + self._get_prefix()
         self._run_cmd(cmd, env=env)
 
+class CMakeBootStrapBuilder(MakeBuilder):
+    def _build(self, install_dirs, reconfigure):
+        env = self._compute_env(install_dirs)
```

Comment:
Is this meant to be passed somewhere / used?

Reply:
my bad, this can be deleted.